quiz.js

```diff
@@ -37,7 +50,20 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'House Stark'}, {$push: {member: {name: 'Ned Stark'}}});
+
+db.houses.update({
+  name: 'House Stark'}, {$push: {member: {name: 'Arya Stark'}}});
+
+db.houses.update({
+  name: 'House Stark'}, {$push: {member: {name: 'Sansa Stark'}}});
 
+db.houses.update({
+  name: 'House Targaryen'}, {$push: {member: {name: 'Daenerys Targaryen'}}});
+
+db.houses.update({
+  name: 'House Arryn'}, {$push: {member: {name: 'Jon Arryn'}}});
 
```

Comment:
This will work, but since you're doing an update, one thing you could have done to make this even shorter is to use [$each](http://docs.mongodb.org/manual/reference/operator/update/each/), like so:

```
db.houses.update({name: 'House Stark'}, {$push: {member: {$each: 
  [ 
    {name: 'Ned Stark'},
    {name: 'Arya Stark'},
    {name: 'Sansa Stark'}
  ]
}}})
```